build/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Any reason for adding all the default react icon set and config?

Reply:
The default icon set and `manifest.json` enable PWA functionality, ensuring proper display across devices, customizable theming, and future-readiness for app installation.